ui/demo/less/colors.less

```diff
@@ -5,3 +5,5 @@
 
 @blue: #1890ff;
 @light-blue: #40a9ff;
+@initial-blue: #1857B6;
+@hover-active-blue: #5492EF;
```

Comment:
I wonder if these values could be imported from the library since you have them defined in `styles.less` as well.

Reply:
Or removed altogether if it turns out you don't need them in `reader.less`.

Reply:
@carolinepaulic im not sure about this either we may need to experiment with this